CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Change Log
 
+## [v0.10.0](https://github.com/SeldonIO/alibi-detect/tree/v0.10.0) (2022-06-01)
+[Full Changelog](https://github.com/SeldonIO/alibi-detect/compare/v0.10.0rc1...v0.10.0)
+
+### Added
+- **New feature** TensorFlow is now an optional dependency. Error messages for incorrect use of detectors that are dependent on missing optional dependencies have been improved to include installation instructions and be more informative.
+
+### Changed
+- The optional dependency work has resulted in some imports being reorganised. This doesn't affect legacy imports as long as the installed dependencies are the same. ([#538](https://github.com/SeldonIO/alibi-detect/pull/538)).
```

Comment:
Nitpick - Would it make more sense to say "The original imports will still work as long as the relevant optional dependencies are installed."? Wondering if the meaning of "legacy imports" is obvious, and if "installed dependencies are the same" is clear?

Reply:
Yep makes sense